editor/src/Components/DataTable.js

```diff
@@ -33,13 +34,22 @@ const DataTable = ({
   const roles = getRoles();
   const isReadOnlyUser = isReadOnly(roles);
 
+  // Sets the state of the value for the current field being edited
+  const [editValue, setEditValue] = useState();
```

Comment:
eslint is barking because `null`/`undefined` is not a valid value to pass to `input` elements. we need an empty string as the default value and whenever you reset the value.

Reply:
i'm also getting this warning when i start editing a blank field:
> Warning: A component is changing an uncontrolled input of type text to be controlled.

i _think_ this might resolve itself if you fix make default input type a string.